src/app/modules/consultations/consultation-list/consultation-list.graphql.ts

```diff
@@ -1,4 +1,18 @@
 import gql from 'graphql-tag';
+import { environment } from '../../../../environments/environment';
+//TODO: Profanity filter feature, remove condition when ready fo deployment to production
+const fragments = {
+  locationId: !environment.production ? gql `
+    fragment locationId on Ministry {
+      name
+      locationId
+    }
+  `: gql `
+    fragment locationId on Ministry {
+      name
```

Comment:
Not clear why `name` is moved to the fragment when it is present on both prod and staging

Reply:
At least one field is required or we will get an error in production as no field will be there.

Reply:
@Skt2021 This is not clear to the person working on the code. Please add comments in cases like this explaining why certain things are being done the way they are

Reply:
Done